src/Sarif/IFileSystem.cs

```diff
@@ -217,5 +217,11 @@ public interface IFileSystem
         /// The name of the empty directory to remove. This directory must be writable and empty.
         /// </param>
         void DeleteDirectory(string path, bool recursive = false);
+
+        /// <summary>
+        /// Deletes a file from a specified path.
```

Comment:
>Deletes [](start = 12, length = 7)

I have been copying this documentation from Microsoft online docs, here: [https://docs.microsoft.com/en-us/dotnet/api/system.io.file.delete?view=netcore-3.1](https://docs.microsoft.com/en-us/dotnet/api/system.io.file.delete?view=netcore-3.1)

So, summary: "Deletes the specified file.", param: "The name of the file to be deleted. Wildcard characters are not supported."